README.md

```diff
@@ -363,24 +387,16 @@ augur export v2 \
   --metadata results/subsampled_metadata.tsv \
   --color-by-metadata country \
   --geo-resolutions country \
-  --output auspice/ncov.json
-```
-
-To visualize the final tree, we can drag its JSON file on to the [auspice.us](http://auspice.us) landing page. If you are working from a Binder interface, download the JSON file in the `auspice/` directory to your computer, open a file explorer/finder window, and drag the file on to the auspice.us interface.
-
-If you are working from a GitPod interface or have Auspice installed locally, you can run a local Auspice server with the following command.
-
-``` bash
-auspice view --datasetDir auspice/
+  --output auspice/nextstrain-walkthrough.json
 ```
 
-Then, you can view the trees by navigating to http://localhost:4000 or the corresponding address from GitPod.
+View the tree with additional metadata in Auspice. It should look something like below. Note the newly added info.
```

Comment:
there's no "below"?

Reply:
Oops. I had written the text part of 96f464c536b11b77c838d2d3820b317fa4081e16 and started adding screenshots, then decided I should post this for review first before finishing that commit. I'll still wait for more commentary, but the general idea is that there will be screenshots after each `augur export` command. I've added placeholders in d187ba0da5246af45e8e9cce1f2b2dad38259366 so that work in progress is more obvious.